internal/cli/atlas/security/ldap/save.go

```diff
@@ -96,22 +96,28 @@ func (opts *SaveOpts) validate() error {
 	return nil
 }
 
-func (opts *SaveOpts) newLDAPConfiguration() *atlas.LDAPConfiguration {
-	ldapConfig := &atlas.LDAPConfiguration{
-		LDAP: &atlas.LDAP{
+func (opts *SaveOpts) newLDAPConfiguration() *atlasv2.UserSecurity {
+	ldapConfig := &atlasv2.UserSecurity{
```

Comment:
Now I see naming issue. This looks quite trivial as there are +5 NDS models where we can replace NDS with security. For time being you can do that inside the transformation engine. Worth trying?

Reply:
We could do this as part of the naming conventions proposal in the future I think. Might not be a priority for now as these names are also present in the public OpenAPI spec